### DISCUSSION TITLE
how do you get better topics for projects

### DISCUSSION THREAD

**Discussion by RanvilleCarlo** [community/community#68987] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I want to make a database management system project to store the employee details in the database add a delivery option for the producers to be delivered and maintain the employee details and order details in the company database management system

**Comment by joywin2003** [comment]
checkout this repository https://github.com/anu0012/awesome-computer-science-opportunities, it has alot of topics under computer science and you choose whichever fits you and also you can use chatpgt to find some topics
